nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/SchemaManager.java

```diff
@@ -43,20 +43,22 @@
 
     Schema getSchemaFromPrefix(String schemaPrefix);
 
+    /**
+     * @deprecated since 11.1, seems unused
+     */
+    @Deprecated(since = "11.1")
```

Comment:
wdyt between `@Deprecated(since = "11.1", forRemoval = true)` (which i used) vs `@Deprecated(since = "11.1")`  to be consistent with the rest of code.

Reply:
I don't state `forRemoval` because it's our only case. Don't have any arguments against one or other solution, TBD.